src/consts.js

```diff
@@ -94,23 +116,24 @@ const pubConsts = {
   MD_PERMISSION_EMPTY: 0,
 };
 
-const LIB_FILENAME = {
+const getLibFilename = () => ({
   win32: path.join('./', libLocationModifier, 'safe_app.dll'),
```

Comment:
@bochaco, while this PR works, I think actually it's simpler if we remove this `libLocationModifier` and make this purely the filename as it was previously, and then force the `customLibPath` to include `prod` or `mock`.

That would mean some smaller rewrite of apps, but would be cleaner in the long run (as opposed to this 'opt-in' approach)....

Thoughts?

Reply:
I definitely prefer PR #222